### PR TITLE
providers/heroku: Documentation improvements

### PR DESCRIPTION
**Pull request by justincampbell**

### CODE REVIEW

File: website/source/docs/providers/heroku/r/addon.html.markdown [hunk 1]
```diff
@@ -19,8 +19,14 @@ resource "heroku_app" "default" {
...
 
+# Create a database, and configure the app to use it
+resource "heroku_addon" "database" {
+  app = "${heroku_app.default.name}"
+  plan = "heroku-postgresql:hobby-basic"
+}
+
 # Add a web-hook addon for the app
-resource "heroku_addon" "webhook" {
+resource "heroku_addon" "database" {
```
Comment (phinze) [inline website/source/docs/providers/heroku/r/addon.html.markdown]: This should stay "webhook" yeah?

Comment (justincampbell) [inline website/source/docs/providers/heroku/r/addon.html.markdown]: Yup, thanks! Fixed.

**Comment by catsby** [comment]
:+1:

**Comment by phinze** [comment]
LGTM! I'll leave the merge button to you @justincampbell :grinning:

**Pull request merged**